Product/event-director.md

```diff
@@ -0,0 +1,119 @@
+- [The Event Director](#the-event-director)
+  - [Event Director Context](#event-director-context)
+    - [2. Evaluate Transaction](#2-evaluate-transaction)
+      - [Payload:](#payload)
+    - [2.1. Read Network Map](#21-read-network-map)
+    - [2.2. Determine Rules](#22-determine-rules)
+    - [2.3 Prune the Network Map](#23-prune-the-network-map)
+    - [2.4. De-duplicate Rules](#24-de-duplicate-rules)
+    - [3. Evaluate Transaction](#3-evaluate-transaction)
+      - [Payload](#payload-1)
+
+
+# The Event Director
+
+![tazama-context-event-director](../images/tazama-context-event-director.png)
+
+The foundation of the Tazama Transaction Monitoring service is its ability to evaluate incoming transactions for financial crime behaviors through the execution of a number of conditional statements (rules) that are then combined into typologies that describe the nature of the financial crime that the system is trying to detect. 
+
+
+The Event Director is responsible for determining which typologies a transaction must be submitted to for the transaction to be evaluated. As part of this process, the Event Director determines which rules must receive the transaction and then which typologies are to be scored. The Event Director then routes the transaction to the individual rule processors.
+
+## Event Director Context
+
+```mermaid
+sequenceDiagram
+  participant tmsapi as TMS API
+  participant Event Director as Event Director
```

Comment:
abbreviate as "ed" - not sure how Mermaid will work if the "variable" has a space in it:

  participant ed as Event Director

Remember to carry the variable name down into the body of the flow below as well.